geometry/path.cc

```diff
@@ -197,9 +197,15 @@ bool Path::UpdateMoveComponentAtIndex(size_t index,
 Path::Polyline Path::CreatePolyline(
     const SmoothingApproximation& approximation) const {
   Polyline polyline;
-  auto collect_points = [&polyline](const std::vector<Point>& collection) {
-    polyline.points.reserve(polyline.points.size() + collection.size());
-    polyline.points.insert(polyline.points.end(), collection.begin(),
+  bool new_contour = true;
```

Comment:
Instead of doing this, what if we just made all the `CreatePolyline` calls only return the second point, and made `moveTo` calls use the destination point? I think that reads as slightly more straightforward, but maybe I'm missing something.

Reply:
Including the first point of the next contour also needs to happen if the contour doesn't start with a MoveTo.

Reply:
I think Skia does move to insertion for that but this is probably fine too

Reply:
That can only happen if it's the first contour on the path... lemme see if we're behaving the same as Skia in that case. I think it should start at 0, 0.

Reply:
I like the idea of inserting a move at the beginning and started implementing it, but would prefer to land in a followup if you don't mind. I gotta make it overwrite back to back moves and stuff.

Reply:
SGTM